webapp/tests/karma/unit/services/db-sync.js

```diff
@@ -256,39 +259,24 @@ describe('DBSync service', () => {
 
       let count;
       let retries;
-      const recursiveOnTo = sinon.stub();
-
-      const replicationResultTo = () => {
-        if (count <= retries) {
-          count++;
-          return Q.reject({ code: 413 });
-        } else {
-          return Q.resolve();
-        }
-      };
```

Comment:
When I added the new `on('change)`  event, this test started failing. 
Turned out `replicationResultTo()` was called for every `on` instead of once per `to`, so it counted an extra "try" because of my new listener.